docs/src/pages/demos/steppers/steppers.md

```diff
@@ -5,7 +5,7 @@ components: MobileStepper, Step, StepButton, StepConnector, StepContent, StepIco
 
 # Steppers
 
-<p class="description">Steppers convey progress through numbered steps.</p>
+<p class="description">Steppers convey progress through numbered steps. It provides a wizard-like workflow.</p>
```

Comment:
"wizard" is searched on Algolia but has no match. I have copied the description from the angular material stepper page.

Reply:
Looks good.